tools/static-assets/server/npm-rebuild-args.js

```diff
@@ -13,6 +13,13 @@ var args = [
   "--update-binary"
 ];
 
+// Allow additional flags to be passed via the $METEOR_NPM_REBUILD_FLAGS
+// environment variable.
+var flags = process.env.METEOR_NPM_REBUILD_FLAGS;
```

Comment:
Is it possible that --no-bin-links or --update-binary could ever be problematic? Maybe the env var should replace them rather than append to them?  And then maybe you should be able to explicitly set empty string to have it do something (or I guess you could set it to a space or something).